src/services/rates/currencyConverter.test.ts

```
@@ -0,0 +1,312 @@
+      (prisma.acbuRate.findFirst as jest.Mock).mockResolvedValue({
+        acbuNgn: new Decimal("1000.00"),
+        acbuUsd: new Decimal("0.50"),
+        timestamp: new Date(),
+      });
+
+      const result = await convertLocalToUsd(1000000000, "NGN");
+
+      expect(result).toBeCloseTo(500000, 2);
+    });
+
+    it("should handle decimal input strings", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)